filament/backend/src/metal/MetalBlitter.mm

```diff
@@ -91,12 +91,14 @@
 {
     FragmentOut out = {};
 
+#if defined(BLIT_COLOR) || defined(BLIT_DEPTH)
```

Comment:
What if `defined(BLIT_COLOR) || defined(BLIT_DEPTH)` is true but `defined(MSAA_COLOR_SOURCE) || defined(MSAA_DEPTH_SOURCE)`? Wouldn't the warning still occur?

Reply:
I misread the if-else, sry